modules/nodes-pods-vertical-autoscaler-using-about.adoc

```diff
@@ -160,7 +160,9 @@ spec:
 
 [NOTE]
 ====
-There must be operating pods in the project before the VPA can determine recommended resources and apply the recommendations to new pods.
+Before a VPA can determine recommendations for resources and apply the recommended resources to new pods, operating pods must exist and be running in the project. 
+
+To obtain the most accurate recommendations from the VPA, assess the workload volume deployed on your infrastructure and the activity level of the workload. For example, if a workload consumes a large amount of resources, remains close to its original storage size, and continues to actively manage pods, the VPA might stabilize after a few minutes. Consider that an idle workload could extend the time needed for the VPA to stabilize. 
```

Comment:
This line doesn't make sense to me.  Of note:
* VPA does not manage storage. Why is it mentioned?
* Whether a workload is idle or not doesn't affect how long it takes for VPA to stabilize its recommendation
* Whether a workload consumes a large or small amount of resources does not affect the accuracy of recommendations
* I don't know what it means for a workload to actively mange pods or what that might have to do with VPA.

This is the point we need to get across:
If the workload's resource usage (CPU & memory) looks like it always looks, the VPA's recommendation will be useful within a few minutes.
If the workload's resource usage looks one way at one time, then later looks another way, the VPA will need to collect metrics in both situations in order to make a good recommendation.

Reply:
Hello @joelsmith . How about I remove this note and replace it with:

> If a workload's resource usage, such as CPU and memory is consistent, the VPA can determine recommendations for resources in a few minutes. If a workload's resource usage is inconsistent, unpredictable, the VPA must collect metrics at various resource usage intervals for the VPA to make an accurate recommendation. 

WDYT?

Reply:
That sounds great, thanks!